streaming-mqtt/python/mqtt.py

```diff
@@ -38,19 +38,26 @@ def createStream(ssc, brokerUrl, topic,
         :param storageLevel:  RDD storage level.
         :return: A DStream object
         """
+        jlevel = ssc._sc._getJavaStorageLevel(storageLevel)
+
         try:
-            helper = ssc._jvm.org.apache.spark.streaming.mqtt.MQTTUtilsPythonHelper()
-        except TypeError as e:
-            if str(e) == "'JavaPackage' object is not callable":
+            helperClass = ssc._jvm.java.lang.Thread.currentThread().getContextClassLoader() \
```

Comment:
After 07cb323e7a128b87ef265ddc66f033365d9de463 in Spark you _should_ be able to skip this hack and instead replace with the standard _jvm reference and then in your exception replace the search for classnotfoundexception with "'JavaPackage' object is not callable" inside of TypeError exception.